cypress/integration/eholdings/eholdings-packages.spec.js

```diff
@@ -0,0 +1,41 @@
+/// <reference types="cypress" />
+
+import { testType, feature } from '../../support/utils/tagTools';
+import TopMenu from '../../support/fragments/topMenu';
+import eHoldingsPackages from '../../support/fragments/eholdings/eHoldingsPackages';
+import eHoldingsTitle from '../../support/fragments/eholdings/eHoldingsTitle';
+import eHoldingSearch from '../../support/fragments/eholdings/eHoldingsSearch';
+import eHoldingsPackagesSearch from '../../support/fragments/eholdings/eHoldingsPackagesSearch';
+import eHoldingsPackage from '../../support/fragments/eholdings/eHoldingsPackage';
+
+
+describe('eHoldings packages management', () => {
+  let userId = '';
+  beforeEach(() => {
+    cy.createTempUser(['eHoldings: Can edit providers, packages, titles detail records',
```

Comment:
Maybe it would be good to create some sort of separate file with constants for specific permissions (ui names) and export it into tests when needed?

Reply:
I will try to do this through not gui names. It's shorter. Thank you

Reply:
done